packages/clay-pagination/src/index.tsx

```diff
@@ -157,7 +105,7 @@ const ClayPagination: React.FunctionComponent<IProps> = ({
 						href={
 							hrefConstructor && hrefConstructor(page as number)
 						}
-						key={page as number}
+						key={`page${index}` as string}
```

Comment:
I think it would be better for the key to be `page`. It may happen that `page${index}` change when any item becomes a DropDown, page does not.

Reply:
What do you think about `paginationItem${index}`?

Reply:
Solved internally :)